SIL.DblBundle.Tests/DBLMetadataTests.cs

```diff
@@ -113,23 +113,46 @@ public void GetDateArchived()
 		}
 
 		[Test]
-		public void Load_Successful()
+		public void Load_FromValidFile_Successful()
 		{
 			using (var metadataFile = new TempFile())
 			{
 				File.WriteAllText(metadataFile.Path, Resources.metadata_xml);
-				Exception exception;
-				DblTextMetadata<DblMetadataLanguage>.Load<DblTextMetadata<DblMetadataLanguage>>(metadataFile.Path, out exception);
+				DblTextMetadata<DblMetadataLanguage>.Load<DblTextMetadata<DblMetadataLanguage>>(metadataFile.Path, out var exception);
 				Assert.Null(exception);
 			}
 		}
 
+		[Test]
+		public void Load_FromTextReader_Successful()
+		{
+			using (var textReader = new StringReader(Resources.metadata_xml))
+			{
+				var data = DblTextMetadata<DblMetadataLanguage>.Load<DblTextMetadata<DblMetadataLanguage>>(textReader, "Resources.metadata_xml", out var exception);
+				Assert.Null(exception);
+				Assert.IsNotNull(data);
```

Comment:
To be symmetric, this should be 
```suggestion
				Assert.NotNull(data);
```

(`Assert.IsNotNull` seems to be a NUnit 2 thing - it's not listed in the NUnit 3 documentation)